apps/opentelemetry/otel-collector.yaml

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
We store only applications that are based on NSM SDK in the folder apps/. Please move this near the example.